### DISCUSSION TITLE
What the "Allow iOS Static Frameworks" means in the installation docs?

### DISCUSSION THREAD

**Discussion by ftzi** [invertase/react-native-firebase#4267] [Q&A] [answered]
I just started to make my App also work for iOS, and went to the docs to see what I need to do to make this lib work. I don't know too much about Apple so I am kinda lost here.

What does the "Allow iOS Static Frameworks" section means in the docs? How do I know if I am using static frameworks or not; how do I know if I have to execute this step?

Another lib I am using seems to requires "use_frameworks!" in the PodFile. Is this related to it?

https://rnfirebase.io/#allow-ios-static-frameworks

Thanks and sorry for my ignorance about it.

**Comment by mikehardy** [comment, marked as answer]
I believe you will have a build failure if you require it and don't use it. Which is to say: try it and see
Normally people that need that know they need it because the requirement is documented in the requiring-library's documents. We just support it here as it affects any Objective-C library in your project when you need to flip it on

I'm not the strongest at iOS either but I believe it's related to including Swift frameworks etc
